README.md

```diff
@@ -1,21 +1,28 @@
 # Description #
 
-This repository contains the Multi-UE Proxy to allow UEs to communicate with a single eNB using the customzied OpenAir Interface (OAI) software.
+This repository contains the Multi-UE Proxy to allow UEs to communicate with a single eNB (aka, lte mode), a single gNB (aka, nr mode), or both of them (aka, nsa mode) using the customzied OpenAir Interface (OAI) software.
```

Comment:
This repository contains the Multi-UE Proxy to allow UEs to communicate with a single eNB (LTE mode), or both an eNB and gNB (NSA mode) using the customized OpenAirInterface (OAI) software. Standalone (NR mode) is still under development in the OAI develop branch. When this development is complete this Multi-UE Proxy will offer NR support as well.